src/Microsoft.AspNet.Mvc.Core/OptionDescriptors/IOptionDescriptor.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Mvc.OptionDescriptors
+{
+    /// <summary>
+    /// Encapsulates information that describes a <typeparamref name="TOption"/> option on <see cref="MvcOptions"/> .
+    /// </summary>
+    /// <typeparam name="TOption">The type of the option.</typeparam>
+    public interface IOptionDescriptor<out TOption>
```

Comment:
to confirm: this interface exists only because we prefer not to extend the `OptionDescriptor<TOption>` class?  is it buying us anything else?